src/cardboard-distorter.js

```diff
@@ -98,8 +98,7 @@ function createRenderTarget(renderer) {
   var parameters = {minFilter: THREE.LinearFilter,
                     magFilter: THREE.LinearFilter,
                     format: THREE.RGBFormat,
-                    stencilBuffer: false,
-                    depthBuffer: false};
```

Comment:
Same fix for #57 and #55, just remove the `depthBuffer: false`, i.e. revert https://github.com/borismus/webvr-boilerplate/commit/32cd6f543534960a104e2df9d247c451b74f9e01